src/adaptors/merkl/index.js

```diff
@@ -11,6 +11,21 @@ const networks = {
   60808: 'bob',
 };
 
+// Protocols that should not be listed under Merkl
+// as they already have their own adapters.
+const protocolsBlacklist = [
+  'euler',
+  'crosscurve',
+  'aerodrome',
+  'gamma',
+];
+
+// Allow specific pools from blacklisted protocols
+const poolsWhitelist = [
+  // Pool from Aerodrome CL: xPufETH-WETH
+  '0xCDf927C0F7b81b146C0C9e9323eb5A28D1BFA183',
```

Comment:
Aerodrome has an adapter but not Aerodrome CL
This pool is already listed so it's best to keep it